### ISSUE TITLE
Turn off tracking automatically

### ISSUE THREAD

**Issue by tmos** [redfish64/TinyTravelTracker#17]
Hi,
Would be great to be able to turn off tracking automatically, for exemple bettwen 10pm and 8am, or in certain locations.
And maybe a new entry in the menu "don't track me for..." and a popin where you chose the duration (don't track me for 1/2/4/8/12/24h).

Great job though :)

**Comment by dreaming-augustin** [comment]
I don't think this app is being actively developed. The original developer currently has other priorities, which is understandable, and no other developer has stepped forward and offered a patch. :(

But for the record, this is a very good feature request. I have been thinking along similar lines for a while.

Even for people who leave the tracker on 24/7 during their daily life, there is no need to keep the tracker on when being trapped a work in a building where there is no GPS reception, or at home at night, when we spend 8 hours straight lying in a bed. 
The question is how to automate or make as painless as possible turning tracking on and off?
I don't see how a fully automatic solution would work. How would the app know when we leave a place if it was turned off when we arrived there?

I think maybe the best and the easiest to implement would be an applet (widget) that we could easily place on our home screen, allowing us to toggle tracking on/off very easily and quickly. 

tmos' suggestions are very good. The best would be to provide a combination of all this, allowing the users to chose the best solution according to their situations and living/travelling habits.

**Comment by redfish64** [comment]
I like the "don't track me..." bit.

As far as not recording gps positions while you're sleeping or at work, I
think it's a little bit too complex, and you never know if you might be
woken up in the middle of the night for something...

I was thinking that maybe if the accelerometer shows no movement since the
last few times TTT woke up, ie it's rotational position is the same as
before, its stable, the last few points are somewhat close together, then I
could skip trying the gps and just use the average of the last few points.
What do you think of that? BTW, are you trying to save battery, number of
points in the log, or what? I have 450,000 so far after 5 years of
continuous use and TTT still responds very fast.

Anyway, I'll try and make little improvements over time. Right now I'd
doing my taxes and also am heavily involved with NomicCoin (silly, I know,
but it's fun to do). No guarantee, though. You are welcome to fork/submit
patches.

**Comment by tmos** [comment]
Hi :)
Thanks for answer and the other fix !

I think the easier solution for you will be the best. An option that don't track for X hours will fit perfectly ;)

I'm trying to save battery actually yeah.

**Comment by tmos** [comment]
And I'm à front end dev / ux designer, I'm really poor in java and Android dev. But I can't submit logo and UI parts if you want ?!
